includes/html/print-map.inc.php

```diff
@@ -14,7 +14,8 @@
 
 use LibreNMS\Config;
 
-$highlight_node = $vars['highlight_node'] | 0;
+$highlight_node = $vars['highlight_node'] ?: 0;
```

Comment:
This should actually be ?? :D

Reply:
as you see, it was not :-D

Reply:
?: is the same as `if ($a) $a else $b`
?? is `if isset($a) $a else $b`

Obviously bitwise or is a little wonky.